backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -222,13 +223,16 @@ def _generate_privilege_record(
         jurisdiction_postal_abbreviation: str,
         license_expiration_date: date,
         compact_transaction_id: str,
+        original_issuance_date: date | None = None,
     ):
+        today = datetime.now(tz=UTC).date()
         privilege_object = {
             'providerId': provider_id,
             'compact': compact_name,
             'jurisdiction': jurisdiction_postal_abbreviation.lower(),
             'status': 'active',
-            'dateOfIssuance': datetime.now(tz=UTC).date(),
+            'dateOfIssuance': original_issuance_date if original_issuance_date else today,
+            'dateOfRenewal': today,
             'dateOfExpiration': license_expiration_date,
             'compactTransactionId': compact_transaction_id,
         }
```

Comment:
Accessibility: 
* This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
* This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.

(For both expiration date boxes.)
<img width="208" alt="image" src="https://github.com/user-attachments/assets/eaf90403-53a5-4617-b297-4621306fc064">